homeassistant/setup.py

```diff
@@ -22,11 +22,7 @@
 DATA_DEPS_REQS = "deps_reqs_processed"
 
 SLOW_SETUP_WARNING = 10
-
-# Since its possible for databases to be
-# upwards of 36GiB (or larger) in the wild
-# we wait up to 3 hours for startup
-SLOW_SETUP_MAX_WAIT = 10800
+SLOW_SETUP_MAX_WAIT = 180
```

Comment:
There are some integrations that actually take 5 minutes to setup still.  I believe the I-zone thermostats will fail if its this low.